src/components/schemas/Rental.yaml

```
@@ -0,0 +1,9 @@
+type: object
+properties:
+  monthlyPrice:
+    type: number
+    format: double
+    minimum: 10
+    maximum: 100000
+  yield:
+    $ref: ./Yield.yaml
```

suggestion: Ajouter un saut de ligne à la fin du fichier.
```suggestion
    $ref: ./Yield.yaml

```